ClassMoodApp/static/js/controllers.js

```diff
@@ -73,9 +74,9 @@ app.controller('dialogController', function($scope, $log, $http) {
         var className = $scope.classname;
         var studentID = 1;
         if (className !== '') {
-            $http.post('/set_student_lectures/' + className + '/' + studentID)
+            $http.post('/set_student_classes/' + className + '/' + studentID)
             .then(function successfulCallback(response) {
-                if (response.data === true) {
+                if (response.data == 'true') {
```

Comment:
No this is incorrect, don't use string comparison. Update the API and this function